src/services/TerminologyService.js

```diff
@@ -1,23 +1,18 @@
 import axios from "axios";
 import { getInstanceIdentifier } from "../utils/ethUtils";
-import { defaultTerminologies } from "../utils/terminologies";
 import BaseService, { baseURL } from "./BaseService";
 axios.defaults.baseURL = baseURL;
 
 export default class TerminologyService extends BaseService {
   entity = "terminologies";
 
-  static getSavedTerminologies() {
-    const savedTerminologies = localStorage.getItem("terminologyObject");
-
-    if (!savedTerminologies) return defaultTerminologies;
-
-    return JSON.parse(localStorage.getItem("terminologyObject")).terminologies;
-  }
-
   getCurrentTerminology() {
     return axios.get(`${this.entity}`, { params: { schemeIdentifier: getInstanceIdentifier() } });
   }
+
+  getDefaultTerminologies() {
+    return axios.get(`${this.entity}`, { params: { schemeIdentifier: "default" } });
+  }
```

Comment:
I really don't think this should be being done in two different queries. The endpoint should give you either all of the terms or the ones for default AND a given scheme.

Reply:
I have updated the endpoint accordingly Please check this [backend PR](https://github.com/civicledger/wat-adonis-api/pull/6/files):